src/main/java/teammates/common/util/Const.java

```diff
@@ -1028,7 +1028,7 @@
         public static final String INSTRUCTOR_COURSE_EMPTY = "There are no students in this course. Click <a href=\"%s\">here</a> to enroll students.";
         public static final String INSTRUCTOR_PERSISTENCE_ISSUE = "Account creation is still in progress. Please reload the page"
                 + " after sometime.";
-        public static final String INSTRUCTOR_NO_MODIFY_COURSES_SESSION_PERMISSION = "No permission to modify course's sessions!";
+        public static final String INSTRUCTOR_NO_MODIFY_PERMISSION_FOR_ACTIVE_COURSES_SESSIONS = "No permission to modify course's sessions!";
```

Comment:
I was referring to the variable value rather than name. But my comment applies to both.